tests/test_adapter_pydantic_v1.py

```diff
@@ -0,0 +1,106 @@
+import unittest
+import warnings
+from types import MappingProxyType
+from unittest import mock
+
+from itemadapter.utils import get_field_meta_from_class
+from tests import (
+    AttrsItem,
+    DataClassItem,
+    PydanticV1Model,
+    PydanticV1SpecialCasesModel,
+    ScrapyItem,
+    ScrapySubclassedItem,
+    clear_itemadapter_imports,
+    make_mock_import,
+)
+
+
+class PydanticTestCase(unittest.TestCase):
+    def test_false(self):
+        from itemadapter.adapter import PydanticAdapter
+
+        self.assertFalse(PydanticAdapter.is_item(int))
+        self.assertFalse(PydanticAdapter.is_item(sum))
+        self.assertFalse(PydanticAdapter.is_item(1234))
+        self.assertFalse(PydanticAdapter.is_item(object()))
+        self.assertFalse(PydanticAdapter.is_item(DataClassItem()))
+        self.assertFalse(PydanticAdapter.is_item("a string"))
+        self.assertFalse(PydanticAdapter.is_item(b"some bytes"))
+        self.assertFalse(PydanticAdapter.is_item({"a": "dict"}))
+        self.assertFalse(PydanticAdapter.is_item(["a", "list"]))
+        self.assertFalse(PydanticAdapter.is_item(("a", "tuple")))
+        self.assertFalse(PydanticAdapter.is_item({"a", "set"}))
+        self.assertFalse(PydanticAdapter.is_item(PydanticV1Model))
+
+        try:
+            import attrs  # noqa: F401
+        except ImportError:
+            pass
+        else:
+            self.assertFalse(PydanticAdapter.is_item(AttrsItem()))
+
+        try:
+            import scrapy  # noqa: F401
+        except ImportError:
+            pass
+        else:
+            self.assertFalse(PydanticAdapter.is_item(ScrapyItem()))
+            self.assertFalse(PydanticAdapter.is_item(ScrapySubclassedItem()))
+
+    @unittest.skipIf(not PydanticV1Model, "pydantic <2 module is not available")
```

Comment:
would it make sense to skip the whole test module if pydantic1 is not available?

Reply:
For the record, pydantic1 is always available if pydantic (whatever version) is installed.

It may make sense not to run extra-specific tests if the corresponding extra is not installed. But only if the corresponding adapter is not available in that scenario, it currently is available and I think it makes sense to test its “is_item” implementation in scenarios where the corresponding extra is not installed.

I think it may be best to discuss this in a separate issue or PR.

Reply:
Should we release this now or do that separate discussion first?

Reply:
I’m fine with either approach.